src/actors/builtin/account/account_actor.go

```diff
@@ -1,19 +1,72 @@
 package account
 
 import (
+	addr "github.com/filecoin-project/go-address"
 	builtin "github.com/filecoin-project/specs/actors/builtin"
 	vmr "github.com/filecoin-project/specs/actors/runtime"
 	cid "github.com/ipfs/go-cid"
 )
 
 type InvocOutput = vmr.InvocOutput
 
+type AccountActorCode interface {
+	Impl() *AccountActorCode_I
+}
+
+type AccountActorState interface {
+	Address() addr.Address
+	Impl() *AccountActorState_I
+}
```

Comment:
As for #820, make the state a vanilla struct, remove the Code interface, and add an empty struct for the receiver type AccountActor.